solr/core/src/java/org/apache/solr/core/ZkContainer.java

```diff
@@ -219,10 +219,10 @@ public void registerInZk(final SolrCore core, boolean background, boolean skipRe
             } catch (InterruptedException e) {
               // Restore the interrupted status
               Thread.currentThread().interrupt();
-              SolrException.log(log, "", e);
+              log.error("", e);
```

Comment:
```suggestion
              log.error("Interrupted", e);
```

BTW I don't get why we had all these logs with empty strings.  On the consuming side as a user, it's weird to see when these exceptions happen.